webrtc.html

```diff
@@ -720,7 +720,7 @@
                   </li>
 
                   <li>
-                    <p>Fire a simple event named <code><a href=
+                    <p>Queue a task to fire a simple event named <code><a href=
```

Comment:
We are already in a queued task here, so reentering that task is impossible.

Nor is there any risk of garbage collection, since the peer connection is held alive by the stack.

This change alters the order in which a signalingstatechange event-handler and a promise onRejected function are called. Is that desirable?